variables.tf

```diff
@@ -161,6 +174,19 @@ variable "sysdig_tags" {
   default     = []
 }
 
+variable "sysdig_access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the resources created by the module. For more information, see https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial."
```

Comment:
`to the resources created by the module` -> `to the Sysdig instance created by the module`